packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap

```diff
@@ -2064,15 +2082,62 @@ exports[`test file matches snapshot 1`] = `
       "total_size": 1,
       "type": 9,
       "vlen": false,
+      "vlen_type": {
+        "littleEndian": true,
+        "signed": true,
+        "size": 1,
+        "type": 0,
+        "vlen": false,
+      },
     },
     "shape": [],
     "type": {
       "base": {
-        "class": "Unknown",
+        "class": "Integer",
+        "endianness": "little-endian",
+        "size": 8,
       },
       "class": "Array (variable length)",
     },
-    "value": "Uint8Array (unstable)",
+    "value": Int8Array [
+      0,
+      1,
+    ],
+  },
+  {
+    "name": "vlen_float32_1D",
+    "rawType": {
+      "littleEndian": true,
+      "signed": false,
+      "size": 8,
+      "total_size": 2,
+      "type": 9,
+      "vlen": false,
+      "vlen_type": {
+        "littleEndian": true,
+        "signed": false,
+        "size": 4,
+        "type": 1,
+        "vlen": false,
+      },
+    },
+    "shape": [
+      2,
+    ],
+    "type": {
+      "base": {
+        "class": "Float",
+        "endianness": "little-endian",
+        "size": 32,
+      },
+      "class": "Array (variable length)",
+    },
+    "value": [
+      Float32Array [],
```

Comment:
- typed array 🎉 
- empty because no vlen array was explicitly assigned at this index 🎉